packages/@storefront/flux-capacitor/src/core/actions/creators.ts

```diff
@@ -453,6 +453,21 @@ namespace ActionCreators {
     ];
   }
 
+  /**
+   * Selects the given refinements based on navigationId and indices.
+   * @param  {string}                               navigationId - The navigationId for
+   * the navigation to fetch more refinements against.
+   * @param  {number[]}                             indices      - The indices of the refinements
+   * intended to be selected.
+   * @return {Actions.ResetPageAndSelectMultipleRefinements}     - Actions with relevant data.
+   */
+  // tslint:disable-next-line max-line-length
+  export function selectMultipleRefinements(navigationId: string, indices: number[]): Actions.ResetPageAndSelectMultipleRefinements {
+    const actions: Actions.ResetPageAndSelectMultipleRefinements = [ActionCreators.resetPage()];
+    indices.forEach((i) => actions.push(...ActionCreators.selectRefinement(navigationId, i)));
```

Comment:
Since each invocation of `selectRefinement` returns a new `resetPage` action, do we end up with duplicate actions in the resulting array? If so, is this an issue?